inspirehep/base/templates/format/record/Inspire_Default_HTML_brief.tpl

```diff
@@ -36,16 +36,25 @@
               </div>
               <div class="col-md-11">
                 <h4 class="custom-h">
+                  <b>
```

Comment:
There is no closing of this tag. It is not needed actually since the change to Helvetica font.